ImperatorToCK3/Data_Files/configurables/succession_law_map.txt

```diff
@@ -0,0 +1,39 @@
+# IMP-CK3 SUCCESSION LAW MAPPINGS #
+# link = { imp = [law] ck3 = [law] }
+# The CK3 law is output in the title's history
+# folder, right after the line defining
+# the holder, as succession_laws = { [law] }
+
+# CK3 has default succession laws
+
+# For mappings with multiple ck3 laws, all laws will be written out
+
+
+# Monarchy
+
+link = { imp = agnatic_succession_law ck3 = male_only_law ck3 = single_heir_succession_law }
+link = { imp = agnatic_succession_law ck3 = single_heir_succession_law }
+link = { imp = cognatic_succession_law ck3 = male_preference_law }
+link = { imp = agnatic_seniority_succession_law ck3 = single_heir_dynasty_house }
+link = { imp = agnatic_succession_law ck3 = single_heir_succession_law }
```

Comment:
Looks to be a copy of line 15

Reply:
The mappings were provided by @IohannesIohannium so I don't know what should be here. Will remove the duplicate for now.